### PR TITLE
Notification of sync delay

### LINKED ISSUE #289: Add note that apps are synced every "few" minutes.
Toto je celkom dolezita info a asi by som to dal aj niekde k "Save" buttonu

### PR DESCRIPTION
**Pull request by tomasfejfar**
Fixes #289 
Related to #360

### CODE REVIEW

File: src/scripts/modules/home/components/apps/EditAppForm.js [hunk 1]
```diff
@@ -458,6 +458,9 @@ class EditAppForm extends Component {
               >
                 Save
               </button>
+              <p class="help-block small margin-vertical-1em">
+                Changes may take up to 5 minutes to propagate to Keboola Connections.
```
Comment (ondrejhlavacek) [inline src/scripts/modules/home/components/apps/EditAppForm.js]: typo: `Keboola Connection`

Comment (tomasfejfar) [reply src/scripts/modules/home/components/apps/EditAppForm.js]: Fixed

**Comment by ujovlado** [comment]
Thx, I updated original comment a bit. Actually this fixes #289.

**Comment by ujovlado** [comment]
screenshot_2018-02-01_19-44-31

**Pull request merged**